pkg/apis/healthchecking/v1alpha1/machinehealthcheck_types.go

```diff
@@ -6,6 +6,31 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
+const (
+	// EventRemediationRestricted is emitted in case when machine remediation
```

Comment:
I think this should be in the controller instead along with https://github.com/openshift/machine-api-operator/blob/master/pkg/controller/machinehealthcheck/machinehealthcheck_controller.go#L30

Reply:
I wanted to make event types part of the API so anybody can use these constants in their programs. Maybe it was my wrong assumption so I moved constants to controller.